baybe/kernels/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Kernels that can be used for GP surrogate models."""
+
+from baybe.kernels.base import Kernel
```

Comment:
The base class should not be exposed via the init file (see other packages)

Reply:
Changed